packages/subgraph/schema.graphql

```diff
@@ -49,7 +49,7 @@ type Permission @entity(immutable: true) {
   contractPermissionId: ContractPermissionId!
   who: Bytes!
   actor: Bytes!
-  oracle: Bytes
+  condition: Bytes
```

Comment:
looking at the wording "condition" gives the sensation of the condition being known, however it is the address in this case.
should the name be something like: conditionHandler or conditionHandlerAddress, ...etc